odoo/addons/base/ir/ir_cron.py

```diff
@@ -170,105 +177,107 @@ def _process_job(cls, job_cr, job, cron_cr):
             cron_cr.commit()
 
     @classmethod
-    def _acquire_job(cls, db_name):
+    def _process_jobs(cls, db_name):
         """ Try to process all cron jobs.
```

Comment:
Shouldn't the descriptions of `_acquire_job` and `_process_jobs` methods be a bit different?